dev-php/composer/composer-1.2.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit eutils versionator
+
+DESCRIPTION="A dependancy manager for PHP"
+HOMEPAGE="https://getcomposer.org"
+
+SRC_URI="https://getcomposer.org/download/${PV}/${PN}.phar -> ${P}.phar"
```

Comment:
The .phar is a php file that contain some compressed part, it is the format they use to distribute it.
https://secure.php.net/manual/en/intro.phar.php
https://getcomposer.org/download/

Were talking about it on this bug (trying to not use it at all) https://bugs.gentoo.org/show_bug.cgi?id=439206